### PR TITLE
Adding files for #579

### LINKED ISSUE #579: Tweet Scraper with a Particular hashtag using Twint
## **Is your feature request related to a problem? Please describe.**

A Short and Simple Python Script to Scrap Tweets of particular Hashtag **without** Twitter API (Tweepy)

## **Describe the solution you'd like**

Simple Python Script using [Twint](https://github.com/twintproject/twint)

## **Additional context**
Twint ⇒ https://github.com/twintproject/twint 

### **Do I Want to work on this ?** 
*Yes*

### PR DESCRIPTION
**Pull request by RohiniRG**
# Description
 This script makes use of snscrape to scrape tweets associated with a particular hashtag. Snscrape is a python library that scrapes twitter without the use of API keys. Here, we take the input of a hashtag and generate details associated with it (Eg., id, username, date, time, the tweet content, number of replies, likes, retweets, quotes and also its URL) tracing upto the desired number of tweets. We store all this information in a .csv file with the selected file name.

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #579 
Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by Siddhant-K-code — APPROVED** [review]
Good Work!
But, we were going to use Twint Module as per #579

File: Python/Twitter_Hashtags/Hashtag_Scrape.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    with open(file_name+'.csv', 'a', encoding='utf-8') as csv_f:
+        csv_pointer = csv.writer(csv_f, delimiter=',')
+        csv_pointer.writerow(header_list)
+        csv_pointer.writerows(tweets_list)
+
+    print(f'Done! Check your directory for {file_name}.csv file!')
+
+else:
+    print('No tweets available for this hashtag')
```
Comment (Siddhant-K-code) [inline Python/Twitter_Hashtags/Hashtag_Scrape.py:33]: Add a newline at EOF

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]

**Comment by RohiniRG** [comment]
> Good Work!
> But, we were going to use Twint Module as per #579

I thought snscrape would solve the same issue without using any twitter API

**Comment by RohiniRG** [comment]
I added the newline at the eof as mentioned. What am I supposed to do next? I am a little confused as this is my first PR.

File: Python/Twitter_Hashtags/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+.csv file generated accordingly:
+
+![Imgur](https://i.imgur.com/YdGt9z4.png)
+
+## Author
+
+[Rohini Rao!](https://github.com/RohiniRG)
+
+
+
```
Comment (HarshCasper) [inline Python/Twitter_Hashtags/README.md:37]: Remove these lines

File: Python/Twitter_Hashtags/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+![Imgur](https://i.imgur.com/9aK1bhi.png)
+
+.csv file generated accordingly:
+
+![Imgur](https://i.imgur.com/YdGt9z4.png)
+
+## Author
+
+[Rohini Rao!](https://github.com/RohiniRG)
```
Comment (HarshCasper) [inline Python/Twitter_Hashtags/README.md:34]: Remove exclamation mark

File: Python/Twitter_Hashtags/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+2. csv module:
+It comes built in with python
+
+## Working
+
+The inputs user has to give: 
+
+![Imgur](https://i.imgur.com/9aK1bhi.png)
```
Comment (HarshCasper) [inline Python/Twitter_Hashtags/README.md:26]: Use `![image]`

File: Python/Twitter_Hashtags/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+.csv file generated accordingly:
+
+![Imgur](https://i.imgur.com/YdGt9z4.png)
```
Comment (HarshCasper) [inline Python/Twitter_Hashtags/README.md:30]: Use `![image]`

File: Python/Twitter_Hashtags/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+2. csv module:
+It comes built in with python
+
+## Working
+
+The inputs user has to give: 
+
+![Imgur](https://i.imgur.com/9aK1bhi.png)
+
+.csv file generated accordingly:
```
Comment (HarshCasper) [inline Python/Twitter_Hashtags/README.md:28]: Keep .csv in backticks to render it like this: `.csv`

File: Python/Twitter_Hashtags/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+- We store all this information in a .csv file with the selected file name.
+
+## Setup Instructions
+
+### Requirements
+
+1. snscript library: (Supported for Python 3.8 and above only) It's development version is installed as
+
+```
+pip install git+https://github.com/JustAnotherArchivist/snscrape.git
```
Comment (HarshCasper) [inline Python/Twitter_Hashtags/README.md:16]: Is it available on PyPi? We don't need to install git repos and then build it from scratch

File: Python/Twitter_Hashtags/README.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+- In this script, we take the input of a hashtag and generate details associated with it (Eg., id, username, date, time, the tweet content, number of replies, likes, retweets, quotes and also its URL) tracing upto the desired number of tweets. 
+
+- We store all this information in a .csv file with the selected file name.
+
+## Setup Instructions
+
+### Requirements
+
+1. snscript library: (Supported for Python 3.8 and above only) It's development version is installed as
```
Comment (HarshCasper) [inline Python/Twitter_Hashtags/README.md:13]: Is it beneficial for us? Many Users (including me) use either Python 3.6/3.7.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Fix the errors on DeepSource as well, available [here](https://deepsource.io/gh/HarshCasper/Rotten-Scripts/run/ef4e2dee-9d02-48bf-a098-8063f4ca9694/python/)

**Comment by HarshCasper** [comment]
Thanks @Siddhant-K-code for the review as well! Appreciated 🎉

File: Python/Twitter_Hashtags/README.md (hunk 8 above)
Comment (RohiniRG) [reply Python/Twitter_Hashtags/README.md:13]: Snscrape works in python version 3.6 as well, however we cannot have access to replyCount, likeCount and so on, which is available in the development version. For python 3.6, we can get id, url, the actual tweet and username? Should I make changes accordingly then for 3.6 and pypi snscrape?

**Comment by RohiniRG** [comment]
@HarshCasper I have made the updates suitable for the popular python version and snscrape through pypi. Please have a look at and let me know!

File: Python/Twitter_Hashtags/Hashtag_Scrape.py [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (vybhav72954) [inline Python/Twitter_Hashtags/Hashtag_Scrape.py:33]: Please add blank line at EOF

File: Python/Twitter_Hashtags/README.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+- We store all this information in a .csv file with the selected file name.
+
+## Setup Instructions
+
+### Requirements
+
+1. snscrape library:
+
+```
```
Comment (vybhav72954) [inline Python/Twitter_Hashtags/README.md:15]: Can you please fence this code block properly by adding a language attribute?

It should look like [this](https://python-markdown.github.io/extensions/fenced_code_blocks/#:~:text=Fenced%20Code%20Blocks%20are%20defined,on%20a%20line%20by%20themselves.)

Use  shell or bash as the attrciute

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: Python/Twitter_Hashtags/README.md (hunk 8 above)
Comment (HarshCasper) [reply Python/Twitter_Hashtags/README.md]: Its understandable. We can hack our way around this.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**